ports/unixodbc/portfile.cmake

```
@@ -3,16 +3,22 @@ vcpkg_fail_port_install(ON_TARGET "UWP" "Windows")
 vcpkg_from_github(
         OUT_SOURCE_PATH SOURCE_PATH
         REPO lurcher/unixODBC
-        REF 2.3.7
-        SHA512 94e95730304990fc5ed4f76ebfb283d8327a59a3329badaba752a502a2d705549013fd95f0c92704828c301eae54081c8704acffb412fd1e1a71f4722314cec0
+        REF v2.3.9
+        SHA512 473f8d39f5976b4c34394d880d6e511b73e33a1fbd6b828a0929787983cd9b5fe7e16776ed51776ce44b54aa61c62be689c0731489b3989acb99c135fb492ec5
         HEAD_REF master
 )
 
 set(ENV{CFLAGS} "$ENV{CFLAGS} -Wno-error=implicit-function-declaration")
+
+if(VCPKG_TARGET_IS_OSX OR VCPKG_TARGET_IS_LINUX)
+    list(APPEND OPTIONS --with-included-ltdl)
+endif()
+
 vcpkg_configure_make(
         SOURCE_PATH "${SOURCE_PATH}"
         AUTOCONFIG
         COPY_SOURCE
+        OPTIONS ${OPTIONS}
```

```suggestion
vcpkg_configure_make(
    SOURCE_PATH "${SOURCE_PATH}"
    AUTOCONFIG
    COPY_SOURCE
    OPTIONS ${OPTIONS}
```
There should be four spaces.

---

fixed